### ISSUE TITLE
imagick 3.8.0 release on PECL

### ISSUE THREAD

**Issue by tessus** [Imagick/imagick#697]
Can you please release a version on PECL that actually works for PHP 8.3 and 8.4? (For both of these php releases `pecl install imagick` fails.)

@Danack I am more than happy to do it, but I don't have the proper access.

**Comment by jhdxr** [comment]
Hi @Danack may I know if you have any concern for releasing a new version (and if yes, what are they?).

Actually there are forks releasing versions for php 8.3/8.4, so stability is kind of tested.

**Comment by Inscure** [comment]
@Danack

**Comment by bukka** [comment]
This is being worked on - progress for the next release can be seen on [develop branch](https://github.com/Imagick/imagick/tree/develop)

**Comment by tessus** [comment]
@bukka thanks, but I was talking about releasing a version that compiles with PHP 8.3 and 8.4.

Personally I don't need it, because I used the source tree, merged one or 2 PRs and compiled the extension manually. But I guess for most people a `pecl install` is the preferred way to install an extension.

The version that is on PECL does not work on any of the current PHP releases (maybe it works on 8.2). So merging 2 PRs, changing an XML file and creating a new PECL release shouldn't take 3 years.

**Comment by bukka** [comment]
Yeah that's why I'm saying that it's in progress and kept this open - I need to make the release first and the will upload it to pecl. ;)

**Comment by Inscure** [comment]
@bukka – thanks for the update! Do you have an estimated timeline for completing the release and uploading it to PECL? Just wondering when we can expect the final version. :)

**Comment by bukka** [comment]
I just requested addition of myself to Imagick maintainers and hopefully next or following week I would like to release 3.8.0RC1. The stable should hopefully come a week or two after that. It means I'm hoping to have stable release done by the end of March.

**Comment by bukka** [comment]
Just an update here that as some of you might have noticed, the 3.8.0RC2 has been released. It's pretty much the same what will be 3.8.0. The only difference will be for Windows where ImageMagick library should get updated. That's the only thing that is currently holding the release. It should be hopefully all ready soon and planning to do the stable 3.8.0 release in two weeks time. In the meantime, feel free to use 3.8.0RC2 .

Everything is also in master so don't use develop as it got a bit diverged and I'm not currently using that branch for further updates.

**Comment by bukka** [comment]
This has been released - https://github.com/Imagick/imagick/releases/tag/3.8.0 . PECL version also updated.
